### ISSUE TITLE
Blood gems not appearing.

### ISSUE THREAD

**Issue by Anteupplaya** [Noxde/Bloodborne-save-editor#15]
ive had this account for a long time and ik ive gotten blood gems for it. but  when using bb save editor the bloodgem list is empty. why would that be?
![image](https://github.com/user-attachments/assets/559c58bf-5a4f-45ef-89c2-10e246106be5)

**Comment by Noxde** [comment]
I just noticed that you are using the 0.1.0 release, that release doesn't have support for gems. use the latest release [0.3.0](https://github.com/Noxde/Bloodborne-save-editor/releases/tag/0.3.0)

**Comment by Anteupplaya** [comment]
> Ref

that worked. just getting the newest  version worked

**Comment by Anteupplaya** [comment]
actually nvm I created this blood gemstone but its not appearing in game 
![image](https://github.com/user-attachments/assets/281810a6-2a64-46c5-82bc-0f02b7840b53)

**Comment by Noxde** [comment]
> actually nvm I created this blood gemstone but its not appearing in game 
> ![image](https://github.com/user-attachments/assets/281810a6-2a64-46c5-82bc-0f02b7840b53)
> 

Check on your storage, the editor doesn't distinguish between gems in the inventory and gems in the storage right now. You might have edited a gem that's in your storage

**Comment by Anteupplaya** [comment]
all of my gems were in my inventory and even then i tried it again with the blood stone gem from the red jeweled brooch and that worked for some reason. so idk where my tempering abyssal blood gem went..

**Comment by Anteupplaya** [comment]
why would that be

**Comment by Noxde** [comment]
> why would that be
> 
Not sure to be honest, I would need to take a look. There might be an id wrong on the editor or something

**Comment by Anteupplaya** [comment]
i cant send the non edited as tbh i just edited my character ad uploaded not making a backup and wdym just the edited one?

**Comment by Anteupplaya** [comment]
i could send the extracted files as im doing that rn

**Comment by Anteupplaya** [comment]
https://discord.com/channels/1130158543237030049/1293098533582536714/1293324964761567355 this is what this link is too 
![image](https://github.com/user-attachments/assets/406adba9-9392-4f76-86cc-379668416399)
 idk if this will work for sending my extacted data ik im character user 0003 because im max lvl and got the blood gem working

**Comment by Noxde** [comment]
> i cant send the non edited as tbh i just edited my character ad uploaded not making a backup and wdym just the edited one?
> 

I meant the file with the abyssal gem. 
If you only have that one please let me know what effects the gem had

**Comment by Anteupplaya** [comment]
i tried sending the user file but github didnt support it

**Comment by Noxde** [comment]
> https://discord.com/channels/1130158543237030049/1293098533582536714/1293324964761567355 this is what this link is too 
> ![image](https://github.com/user-attachments/assets/406adba9-9392-4f76-86cc-379668416399)
>  idk if this will work for sending my extacted data ik im character user 0003 because im max lvl and got the blood gem working

That zip should work

**Comment by Anteupplaya** [comment]
[Uploading dec_SPRJ0005_CUSA00900.zip…]()

**Comment by Anteupplaya** [comment]
there we go
Again User0003

**Comment by Anteupplaya** [comment]
btw on this save file everything exept the gold and blood gem stone should be in the chest

**Comment by Noxde** [comment]
I don't think it actually uploaded, I'll join htos server and try to get it there.

**Comment by Noxde** [comment]
Can you ping me on your thread? or send me the file on dms im @nox_23 on discord. I don't have access to the thread so I can't get the file

**Comment by Noxde** [comment]
Removed the effect from the file, I forgot to close this
